test/fixtures/render/mixin-array-sample.json

```diff
@@ -146,7 +146,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "[\n  \"x\",\n  \"a\",\n  \"b\",\n  \"y\"\n]"
+                              "content": "[\n  \"x\",\n  \"y\"\n]"
```

Comment:
I think this is a regression. The document has an `include` which brings in the `a` and `b` sample values which are being removed.